main.cpp

```diff
@@ -51,89 +51,90 @@ arm.flex();                             // 3)
 arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
-//  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  1)       
+//  Noun:       baby
+//  action 1:   baby cries
+baby.cries();
+//  action 2:   baby eats
+baby.eats();
+//  action 3:   baby sleeps
+baby.sleeps();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       wife
+//  action 1:   wife reads
+wife.reads();
+//  action 2:   wife cooks
+wife.cooks();
+//  action 3:   wife yells
+wife.yells();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       dog
+//  action 1:   dog runs
+dog.runs();
+//  action 2:   dog hides
+dog.hides();
+//  action 3:   dog barks
+dog.barks();
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       postman
+//  action 1:   postman walks
+postman.walks();
+//  action 2:   postman delivers
+postman.delivers();
+//  action 3:   postman sings
+postman.sings();
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       people
+//  action 1:   people vote
+people.vote();
+//  action 2:   people gossip
+people.gossip();
+//  action 3:   people dance
+people.dance();
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       birds
+//  action 1:   birds fly
+birds.fly();
+//  action 2:   birds chirp
+birds.chirp();
+//  action 3:   birds excrete
+birds.excrete();
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       pastor
+//  action 1:   pastor prays
+pastor.prays();
+//  action 2:   pastor preaches
+pastor.preaches();
+//  action 3:   pastor encourages
+pastor.encourages();
 //  8)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  9)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       student 
+//  action 1:   student studies
+student.studies();
+//  action 2:   student learns
+student.learns();
+//  action 3:   student writes
+student.writes();

```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.